src/drivers/imu/invensense/mpu6000/MPU6000.cpp

```diff
@@ -435,6 +437,7 @@ uint8_t MPU6000::RegisterRead(Register reg)
 	cmd[0] = static_cast<uint8_t>(reg) | DIR_READ;
 	set_frequency(SPI_SPEED); // low speed for regular registers
 	transfer(cmd, cmd, sizeof(cmd));
+	px4_udelay(10);
```

Comment:
Not that it really matters, but maybe putting the delay before the transfer makes slightly more sense?

Reply:
Now that you mention it, yes, it makes slightly more sense